org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/validation/ValidatorFragment2.xtend

```diff
@@ -75,18 +77,29 @@ class ValidatorFragment2 extends AbstractInheritingFragment {
 		return new TypeReference(grammar.runtimeBasePackage + '.validation.' + grammar.simpleName + 'ValidatorConfigurationBlock')
 	}
 	
-		/**
+	/**
 	 * @since 2.14
 	 */
 	protected def getAbstractValidatorConfigurationBlockClass(){
 		return new TypeReference("org.eclipse.xtext.ui.validation.AbstractValidatorConfigurationBlock")
 	}
+	
+	/**
+	 * @since 2.14
+	 */
+	protected def getXbaseValidationConfigurationBlockClass(){
```

Comment:
I was wondering if it would be better to follow the other approach and have something similar to getSuperConfigurableIssueCodesProviderClass (e.g., getSuperValidatorConfigurationBlockClass) instead of having getAbstractValidatorConfigurationBlockClass and getXbaseValidationConfigurationBlockClass

Reply:
Would work if could bind an impl of XbaseValidationConfigurationBlockClass class but we have to bind and impl of AbstractValidatorConfigurationBlockClass instead. This is why I did it this way to make a clear difference.

Reply:
OK, I suspected it was something like that :)